src/programs/menu/common.ts

```diff
@@ -0,0 +1,65 @@
+import { UsersOnBreak } from "@yes-theory-fam/database/client";
+import prisma from "../../prisma";
+import {
+  Message,
+  DMChannel,
+  CollectorFilter,
+  MessageReaction,
+  User,
+} from "discord.js";
+import state from "../../common/state";
+import { createYesBotLogger } from "../../log";
+
+export const mainOptionsEmojis = ["👶", "🦥"];
+export const allCollectedEmojis = ["👶", "🦥", "✅", "🚫"];
+
+export const logger = createYesBotLogger("programs", "DmMenu");
+
+export const handleError = async (
+  optionsMessage: Message,
+  dmChannel: DMChannel
+) => {
+  removeIgnore(dmChannel);
+
+  await dmChannel.send(
+    "Because of technical reasons I can only wait 60 seconds for a reaction. I removed the other message to not confuse you. If you need anything from me, just drop me a message!"
+  );
+  await optionsMessage.delete();
+};
+
+export const isOnBreak = async (userId: string) => {
+  return await prisma.usersOnBreak.findFirst({ where: { userId } });
+};
+
+export const isCooldownDone = async (
+  userData: UsersOnBreak
+): Promise<boolean> => {
+  const twentyFourHours = 24 * 60 * 60 * 1000;
+  const userCoolDownTime = userData.addedAt;
+  return Date.now() - Number(userCoolDownTime) > twentyFourHours;
```

Comment:
```suggestion
  return Date.now() - userCoolDownTime.getTime() > twentyFourHours;
```
(Assuming userCoolDownTime is `Date`); does the same but is more readable.